modules/ossm-creating-istio-system-project-using-console.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-creating-istio-system-project"]
+= Creating the istio-system project using the web console
+:context: ossm-creating-istio-system-project-using-console
+
+.Prerequisties
+
+
+.Procedure
+
+. In the {ocp-product-title} web console, click *Home* -> *Projects*.
+
+. Click *Create Project*.
+
+. At the prompt, you must enter a name for the project in the Name field. For example, `istio-system`. The Operator deploys Istio to the project you specify. The other fields provide supplementary information and are optional.
```

Comment:
I think is not needed to add this information: `The Operator deploys Istio to the project you specify. The other fields provide supplementary information and are optional.`. In the next section, we already specify that we are going to create the resource in the namespace created before. Regarding the extra fields when you create the resource maybe is better to say this directly in the steps or in a Note in the section. Something like this:
```
Note: the `istio` resource definition contains supplementary information that is optional to be defined
```

Reply:
Made minor change here.